controlplane/pkg/nsmd/nsmd_crossconnect_client.go

```diff
@@ -38,6 +38,12 @@ type nsmMonitorCrossConnectClient struct {
 	dataplanes map[string]*dataplaneCrossConnectInfo
 }
 
+func (client *nsmMonitorCrossConnectClient) ClientConnectionAdded(clientConnection *model.ClientConnection) {
```

Comment:
Please add `error` return type here. Since the function is empty, just return `nil`.
Handel the error accordingly when you call the function.